src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/FiltersConfigurator.php

```
@@ -36,18 +33,20 @@ public function __construct(ConfigurationRegistry $registry)
      */
     public function configure(DatagridConfiguration $configuration)
     {
-        $this->configuration = $configuration;
-        $path = sprintf(self::SOURCE_PATH, self::USEABLE_ATTRIBUTES_KEY);
-        $attributes = $this->configuration->offsetGetByPath($path);
+        $path       = sprintf(self::SOURCE_PATH, self::USEABLE_ATTRIBUTES_KEY);
+        $attributes = $configuration->offsetGetByPath($path);
         $attributes = ($attributes === null) ? [] : $attributes;
 
         $displayedFilters = [];
         foreach ($attributes as $attributeCode => $attribute) {
-            $showFilter        = $attribute['useableAsGridFilter'];
+            if (!$attribute['useableAsGridFilter']) {
```

This will result in a silent failure, why this choice ?